trollsift/tests/unittests/test_parser.py

```diff
@@ -451,3 +451,36 @@ def test_no_match(self, fmt, string):
         """Test cases expected to not be matched."""
         with pytest.raises(ValueError):
             parse(fmt, string)
+
+
+class TestIntegers:
```

Comment:
Does it need to be a class? Do we expect other integer-based tests? Maybe just a function?

Reply:
Sure